image-bundle.yaml

```diff
@@ -40,7 +40,7 @@ labels:
   - name: com.redhat.delivery.operator.bundle
     value: "true"
   - name: com.redhat.openshift.versions
-    value: v4.6,v4.7
+    value: v4.6
```

Comment:
isn't 7.11.1 targeting ocp 4.7?

Reply:
I think we support 4.6 and above? this change supports anything 4.6 and above.

Reply:
ok, thanks :)